tradetrust/open-attestation-api/src/app.js

```diff
@@ -42,20 +43,59 @@ function create(){
     logger.error(err);
   }
 
-  app.post('/document/wrap', async function (req, res){
+  
+  app.post("/document/sign", async function(req,res){
+    console.log('/document/sign')
+    if (req.body.document === undefined){throw new UserFriendlyError('No "document" field in payload');}
+    const document = req.body.document; //should be wrapped already at this point
+
+    var keyPair;
+    if(!process.env.DOCUMENT_STORE_OWNER_PRIVATE_KEY){
+      keyPair = {
+        private: '0x40b639e8fb83afe8ad8d4bb7857e69d039a15c3e476b2d98f346222623420e6f',
+        public: '0x103D912298C89a98Eed323376aa4403b92b28842'
+      }
+    }
+    else{
+      keyPair = {
+        //public: `did:ethr:${publicKey}#controller`,
+        public: process.env.DOCUMENT_STORE_OWNER_PUBLIC_KEY,
+        private: process.env.DOCUMENT_STORE_OWNER_PRIVATE_KEY
```

Comment:
in cloud environment this will be encrypted, needs to still be decrypted as example commented out below